strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/ServerConfigurationController.java

```diff
@@ -249,6 +249,11 @@ public ResponseEntity setServerSettings(@RequestBody ServerSettingsForm serverSe
         configurationManagementService.setPort(serverSettingsForm.getPort());
         configurationManagementService.setInstanceName(serverSettingsForm.getInstanceName());
 
+        if(serverSettingsForm.getMaxFileUploadSizeString() != null){
+            String maxFileUploadSize = serverSettingsForm.getMaxFileUploadSizeString();
+            boolean wasAbleToSet = configurationManagementService.setMaxUploadFileSize(maxFileUploadSize);
```

Comment:
What's the purpose of this `boolean`?

Reply:
I was not sure how to handle the size setting in case someone changes the MuptiPart config, so the handling method can throw exception or error response if the property was not able to be set. So I would like to ask how to process this situation inside controller.

Reply:
Variable removed

Reply:
Thanks! :)